src/main/java/dev/jbang/ModularClassPath.java

```diff
@@ -24,6 +24,7 @@ public ModularClassPath(List<ArtifactInfo> artifacts) {
 	public String getClassPath() {
 		if (classPath == null) {
 			classPath = artifacts	.stream()
+									.filter(it -> !it.asFile().getPath().endsWith(".tar.gz"))
```

Comment:
I was wondering, wouldn't a positive filter testing for .jar be better? There might be all kinds of files as part of artifacts but only JARs can be added to the classpath, right? Or do other options exist?